File: docs/src/testers/functions/invocations/cloud_execution.ts
// @sniptest filename=cloud_execution.ts
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();
const fn = client.NotteFunction({
  function_id: process.env.NOTTE_FUNCTION_ID!,
});

// Run on Notte infrastructure (the default).
const result = await fn.run({ url: 'https://example.com' });
console.log(JSON.stringify({ status: result.status, result: result.result }));

File: docs/src/testers/functions/invocations/cloud_execution.py
# @sniptest filename=cloud_execution.py
# @sniptest show=8-12
import os

from notte_sdk import NotteClient

client = NotteClient()
function = client.Function(os.environ["NOTTE_FUNCTION_ID"])

# Runs on Notte infrastructure
result = function.run(
    url="https://example.com",
    local=False,  # Default
)
